draft-nro-bulk-rdap.md

```diff
@@ -19,21 +19,21 @@ email = "secretariat@nro.net"
 
 .# Abstract
 
-To complement the move from Whois to RDAP for the RIRs (Regional Internet Registries), this document specifies a new
-service named the Bulk RDAP that an RIR can deploy to replace their Bulk Whois service.
+To complement the move from Whois to RDAP for the Regional Internet Registries (RIRs), this document specifies a new
+service, named the Bulk RDAP, that an RIR can deploy in lieu of its Bulk Whois service.
```

Comment:
Echoing Yogesh's comment, I think something like the following might help to head off potential concerns about Whois deprecation:

```
The Registration Data Access Protocol (RDAP) is used by Regional
Internet Registries (RIRs) and Domain Name Registries (DNRs) to
provide access to their resource registration information.  For the
corresponding Whois service, RIRs generally publish the information
in bulk format, for use cases where the standard query service is not
suitable.  This document defines a similar bulk service for use with
RDAP.
```

Reply:
Perfect!